src/glslplugin/lang/parser/GLSLParsingBase.java

```diff
@@ -50,276 +53,31 @@
      */
     protected final PsiBuilder psiBuilder;
```

Comment:
If it is used directly, it can be called just `b` as before. I have renamed it before to this to increase the verbosity of unusual code (it was not to be used directly in typical parsing code).